CONTRIBUTING.md

```diff
@@ -84,14 +84,14 @@ It's key to get familiarized with the style guide and mechanics of Thanos, espec
 
 * Find any directory in your system your want Thanos repo in. e.g `~/Repos` -
   * `cd ~/Repos`
-  * Make sure that the GOBIN, GOPATH and GOPROXY (useful) environment variables are set and that GOBIN is included in your PATH. For example -
+  * Make sure that the GOBIN, GOPATH and GOPROXY (useful) environment variables are set and that GOBIN is included in your PATH. You may use `GOPROXY=https://goproxy.cn,direct` as an alternative if you cannot visit `https://proxy.golang.org`. For example -
```

Comment:
I'm able to access `https://goproxy.cn/` in India but not `https://goproxy.cn,direct` is there a typo? 🤔  If not then it would be nice to mention it explicitly that this alternative is only for China.

Reply:
`GOPROXY=direct` is to configure the go command to download all modules directly from source repositories, set GOPROXY to direct. In this case, direct will take effect when we cannot access the proxy. This is optional for developers. I'll remove the direct to avoid misunderstanding.